README.md

```diff
@@ -52,3 +52,48 @@ calculate weighted LD
     if r2 above 0.05 return else skip.
 
 
+# Implementations
+
+## Python
+
+The default implementation of the WeightedLD algorithm is written in Python, and lives in `./WeightedLD.py`. Full usage instructions can be found by running `python WeightedLD.py --help`
+
+In order to run the program, you must have a python environment where the following packages are available:
+- numpy
+- biopython
+
+## Rust
```

Comment:
I really like how you wrote this, clear as day :)

Reply:
Thanks!